spec/acceptance/api/internal/users_api_spec.rb

```diff
@@ -144,4 +144,14 @@
     end
   end
 
+  delete '/services/:service_id/users/all' do
```

Comment:
I think there's no need for the `/all` part.

Reply:
:+1: